src/environments/array.js

```diff
@@ -617,3 +618,17 @@ defineEnvironment({
     htmlBuilder,
     mathmlBuilder,
 });
+
+// Catch \hline outside array envirnoment
+defineFunction({
+    type: "text", // Doesn't matter what this is.
```

Comment:
Why doesn't it matter?

Reply:
Because there's no `handler` defined, so the type isn't used.  I copy/pasted this comment from `functions/math.js`.